cocos/render-scene/core/pass.ts

```diff
@@ -833,8 +839,18 @@ export class Pass {
     get blocks (): Float32Array[] { return this._blocks; }
     get blocksInt (): Int32Array[] { return this._blocksInt; }
     get rootBufferDirty (): boolean { return this._rootBufferDirty; }
+    /**
+     * NOTE: engineInternal tag cannot only mark rootBufferDirty setter as internal
+     * @engineInternal
+     */
+    public setRootBufferDirty (v: boolean) { this._rootBufferDirty = v; }
```

Comment:
public _setRootBufferDirty (val: boolean) {
        this._rootBufferDirty = val;
    }

There has already been a `_setRootBufferDirty` method.

Reply:
but the native Pass class doesn't bind the `_setRootBufferDirty ` method, so here I mark this as  a TODO item